lib/uplink/object.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// ObjectMeta contains metadata about a specific Object
+type ObjectMeta struct {
+	Bucket   string
+	Path     storj.Path
+	IsPrefix bool
+
+	Metadata map[string]string
+
```

Comment:
`ContentType` field here too.

Reply:
same. seems like that should just be in the Metadata map